src/client/cli/cmd/start.cpp

```diff
@@ -101,6 +112,19 @@ mp::ReturnCode cmd::Start::run(mp::ArgParser* parser)
 
     request.set_verbosity_level(parser->verbosityLevel());
 
+    std::unique_ptr<multipass::utils::Timer> timer;
+
+    if (parser->isSet("timeout"))
+    {
+        timer = std::make_unique<multipass::utils::Timer>(
+            std::chrono::seconds(parser->value("timeout").toInt()), [&spinner]() {
+                spinner.stop();
+                std::cerr << "Timed out waiting for instance to start." << std::endl;
```

Comment:
Oh I just noticed, this should use the command's `cerr`, not the std one (here and in the others).